lib/rucio/api/request.py

```diff
@@ -135,3 +135,22 @@ def get_request_by_did(scope, name, rse, issuer):
     req = request.get_request_by_did(scope, name, rse_id)
 
     return api_update_return_dict(req)
+
+
+def list_requests(src_rse, dst_rse, states, issuer):
+    """
+    List all requests in a specific state from a source RSE to a destination RSE.
+
+    :param src_rse: source RSE.
+    :param dst_rse: destination RSE.
+    :param states: list of request states.
+    :param issuer: Issuing account as a string.
+    """
+    src_rse_id = get_rse_id(rse=src_rse)
+    dst_rse_id = get_rse_id(rse=dst_rse)
+
+    kwargs = {'src_rse_id': src_rse_id, 'dst_rse_id': dst_rse_id, 'issuer': issuer}
+    if not permission.has_permission(issuer=issuer, action='list_requests', kwargs=kwargs):
+        raise exception.AccessDenied('%(issuer)s cannot list requests from RSE %(src_rse)s to RSE %(dst_rse)s' % locals())
+
+    return request.list_requests(src_rse_id, dst_rse_id, states)
```

Comment:
For Multi-VO Rucio you have to do something similar to
https://github.com/rucio/rucio/blob/491263e035db97a84dde783b11f5111692b2c3cb/lib/rucio/api/did.py#L62
Otherwise the scope and account would not be correct for Multi-VO